problems/1.intro.js

```diff
@@ -3,8 +3,9 @@
  * 
  * @returns undefined
  */
-function helloWorld() {
-
+function helloWorld(pursuit) {
+      console.log("hello world")
+      return pursuit;
 }
```

Comment:
no params for this function. And no need to return. functions by default return undefined.